src/App.tsx

```diff
@@ -1,24 +1,83 @@
-import React from 'react';
+import React, { Props } from 'react';
+import Clock from './components/Clock/Clock';
 import './App.scss';
 
-const App: React.FC = () => {
-  window.setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean;
+  clockName: number;
 };
 
+class App extends React.Component<Props<State>, State> {
```

Comment:
I think you can use just `{}`, like that: `class App extends React.Component<{}, State>`